virttest/qemu_monitor.py

```diff
@@ -3390,3 +3390,18 @@ def nbd_server_stop(self):
         cmd = "nbd-server-stop"
         self.verify_supported_cmd(cmd)
         return self.cmd(cmd)
+
+    def set_numa_node(self, numa_opt_type, **kwargs):
```

Comment:
```suggestion
    def set_numa_node(self, option_type, **kwargs):
```
Better not need to declear `numa_` for argument `numa_opt_type` since the argument namespace is `numa`.

Reply:
make sense, update param name from `numa_opt_type` to `option_type`, thanks!